tool/teleport/common/usage.go

```diff
@@ -47,7 +47,12 @@ const (
     --uri="http://localhost:8080" \
     --labels=group=dev
   Same as the above, but the app server runs with "group=dev" label which only
-  allows access to users with the application label "group: dev" in an assigned role.`
+  allows access to users with the application label "group: dev" in an assigned role.
+
+> teleport app start --token=xyz --auth-server=proxy.example.com:3080 \
+    --mcp-demo-server
+  Runs a Teleport demo MCP server from the app server.
```

Comment:
This should probably give a very brief description of what the demo MCP server actually does (functions it provides).